get-coverage-percent/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2021, Yext
```

Comment:
nit: change to 2022

Reply:
updated! also updated the license item to check these